ivy/functional/backends/jax/statistical.py

```diff
@@ -37,7 +37,10 @@ def max(
     return jnp.max(a=jnp.asarray(x), axis=axis, keepdims=keepdims)
 
 
-@with_unsupported_dtypes({"0.4.14 and below": "bfloat16"}, backend_version)
+@with_unsupported_dtypes(
```

Comment:
probably we can revert this change, seems like the old format of passing the decorator also passes the lint tests